site/content/en/docs/concepts/_index.md

```diff
@@ -57,16 +57,47 @@ network throughput between the Pods.
+These checks can be external or internal and determine if additional criteria are met before the Workload is admitted. The Workload is admitted once all its [AdmissionCheckStates](site/content/en/docs/concepts/admission_check.md) are mark `Ready`.
+
+<h4> Example: Provisioning AdmissionCheck </h4>
+
+Earlier, Kueue admissions were mainly based on quota checks- if sufficient quota existed, the workload is admitted. However, just because the quota (logical resource) is available, doesn't mean there are enough actual(physical) compute resources in the cluster to schedule all pods successfully. This is critical in cloud environments where autoscaling provides nodes on-demand. 
+
+[ProvisioningRequest AdmissionCheck](keps/1136-provisioning-request-support/README.md) handles this issue by ensuring that the physical resources can be provisioned before admitting Workloads.
+Kueue's enhanced admission process now consists of two sequential checks before admitting a workload:
+- Quota Validation: It uses Kueue's existing quota management system to check if the workload fits within available cluster quotas. It verifies "logical" resource availability. <br>
+- Capacity Guarantee: It uses Provisioning Request and [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) to ensure actual resources will exist. It verifies "physical" resource availability. <br> The Kueue controller creates a ProvisioningRequest object, attaches PodTemplates from the Workload, applies configuration from ProvisioningRequestConfig and sets owner reference to Workload. <br> Cluster Autoscaler receives ProvisioningRequest, checks actual cluster capacity, triggers scaling if needed and updates ProvisioningRequest status as `Provisioned=true` if capacity guaranteed, `Provisioned=false` if scaling is in progress, and `Failed=true` if cannot provision.
+
+Lets understand this with a real-world usage - GPU Workload:
+
+Scenario: *AI training job needing 16 GPUs*
+
+- Step 1 (Quota Check): ClusterQueue has 32 GPU quota available, since requested is quota available, 16 GPUs are reserved.
+
+- Step 2 (Capacity Check): A ProvisioningRequest is created for 16 GPUs. Cluster Autoscaler checks cloud provider GPU inventory and initiates scaling of 4x GPU nodes (4 GPUs each). It sets `Provisioned=true` when nodes ready.
+
+Kueue sees the `Provisioned=true` proceeds to mark the AdmissionCheck `Ready` and admits workload
+
+Outcome:
+*Job starts immediately with all 16 GPUs available.*
+
+<h4>Failure Handling: </h4>
+If the capacity check fails, incase of temporary issues like cloud capacity shortages, exponential backoff retries is triggered while maintaining the workload's queue position and reserved quota. For permanent failures, the AdmissionCheck is marked Rejected, quota is released, and the workload is evicted, requiring user resubmission.
```

Comment:
Indeed holding reserved quota was what we did in the past, but we don't support it anymore. Currently if the admission check fails we release the quota and requeue the Workload